templates/_components/static/list-events.twig

```diff
@@ -17,11 +18,12 @@
                 {}
             ]
         %}
-    
+
     {% for entry in content %}
         <li class="list-event">
             <div class="list-event__container | flex-grid">
-                <a href="#" class="list-event__media">
+                <a
+                    class="list-event__media" href="#">
```

Comment:
Are line breaks here and above <ul> intentional?

Reply:
Fixed!